src/gp_instance.cpp

```diff
@@ -741,14 +767,36 @@ void GPInstance::SubsplitDAGToDot(const std::string &out_path,
   out_stream.close();
 }
 
+void GPInstance::ExportSpanningTreesWithGPBranchLengths(
```

Comment:
Is this our first use of the word "spanning trees"? 

Spanning tree has a meaning https://en.wikipedia.org/wiki/Spanning_tree

Is the notion that it's a set of trees such that they generate the whole DAG? Perhaps "CoveringTrees"? We can discuss.

Reply:
This is the first usage of the term.  Yes, the goal is to create a Newick can generate the DAG with all edges and vertices, with a minimum set of trees.  I agree, covering trees does sound like a better name for this.